tests/test_workers/test_tasks/test_yarn.py

```diff
@@ -0,0 +1,15 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+from unittest import mock
+
+from cachito.workers.tasks import yarn
+
+
+@mock.patch("cachito.workers.tasks.yarn.nexus.execute_script")
+def test_cleanup_yarn_request(mock_exec_script):
+    yarn.cleanup_yarn_request(42)
+
+    expected_payload = {
+        "repository_name": "some-yarn-repo-42",
```

Comment:
get_yarn_proxy_repo_name and get_yarn_proxy_repo_username are actually being called here, so you should use proper strings for these values here

Reply:
Oh, Looks like the problem with test coverage was resolved after changing strings. Thank you. Not everything is random with unit tests :D